includes/drush/custom/diffy.drush.inc

```diff
@@ -0,0 +1,287 @@
+<?php
+
+/**
+ * @file
+ * Drush commands for the module.
+ */
+
+/**
+ * Implements hook_drush_command().
+ */
+function diffy_drush_command() {
+  $items = [];
+
+  // Command to refresh Diffy token based on the env DIFFY_API_KEY variable.
+  $items['diffy-refresh-token'] = [
```

Comment:
Will we need to setup this DIFFY API KEY into drone server?

Reply:
Exactly. And the key is used to generate the token. Which expires daily.